packages/netlify-cms-backend-github/src/GraphQLAPI.ts

```diff
@@ -214,7 +228,64 @@ export default class GraphQLAPI extends API {
     }
   }
 
-  async getStatuses(sha: string) {
+  async getPullRequests(
+    head: string | undefined,
+    state: PullRequestState,
+    predicate: (pr: Octokit.PullsListResponseItem) => boolean,
+  ) {
+    const { originRepoOwner: owner, originRepoName: name } = this;
+    let states;
+    if (state === PullRequestState.Open) {
+      states = ['OPEN'];
+    } else if (state === PullRequestState.Closed) {
+      states = ['CLOSED', 'MERGED'];
+    } else {
+      states = ['OPEN', 'CLOSED', 'MERGED'];
+    }
+    const { data } = await this.query({
+      query: queries.pullRequests,
+      variables: {
+        owner,
+        name,
+        ...(head ? { head } : {}),
+        states,
+      },
+    });
+    const {
+      pullRequests,
+    }: {
+      pullRequests: {
+        nodes: GraphQLPullRequest[];
+      };
+    } = data.repository;
+
+    const mapped = pullRequests.nodes.map(transformPullRequest);
+
+    return ((mapped as unknown) as Octokit.PullsListResponseItem[]).filter(
+      pr => pr.head.ref.startsWith(`${CMS_BRANCH_PREFIX}/`) && predicate(pr),
```

Comment:
@erquhart `pr.head.ref.startsWith(`${CMS_BRANCH_PREFIX}/`)` was missing from the previous PR.